docs/bootableusb.md

```diff
@@ -31,7 +31,37 @@ For this guide, we will be using Rufus to create our bootable USBs.
 2. Plug in your USB.
 3. Locate and launch Rufus. It should ask you to launch with administrator privileges, select YES.
 4. On the right hand side, there should be a button with the words `SELECT` on it.
-5. A File Explorer window should pop up, find your ISO of choie, then click `Open`.
+5. A File Explorer window should pop up, find your ISO of choice, then click `Open`.
 6. Click `Start`. The default options should work just fine.
 7. Wait for the green bar to say `READY`, then click `Close` and eject your newly made USB
 8. Profit
+
+### Flashing using Balena Etcher (Windows and macOS)
+
+For this guide, we will be using Balena Etcher to create our bootable USB on Windows and macOS
+
+1. Download Balena Etcher from [here](https://etcher.balena.io/)
+2. Plug in your USB
+3. Launch Balena Etcher from applications
+4. Click the big blue "Select" button
+5. A File Explorer window should pop up, find the ISO of your choice, then click `Open`
+6. Select your Flashdrive or Change your Flashdrive
+7. Click On Flash
+8. It may ask for adminstrator privileges, click on `allow`
+9. Profit
+
+### Flashing using Balena Etcher (Linux)
```

Comment:
Can we create a shell script that does everything for you? Idk if that's a thing or not.

Reply:
definitely is a thing

Reply:
deleted balena etcher and added a reccomendation for linux and macos with ventoy